scripts/error_on_dirty.sh

```diff
@@ -10,6 +10,7 @@ dirty=$(git ls-files --modified)
 if [[ $dirty ]]; then
     echo $MSG
     echo $dirty
+    git diff
```

Comment:
I use Apple Silicon mac, that's why script for formatting code doesn't work locally for me. It helped me to understand what CI wants from me to be formatted.